### PR TITLE
IDE-850 Perform DNS Lookup on all embedded web pages

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@dehilsterlexis please review

**Comment by dehilsterlexis** [comment]
@GordonSmith @richardkchapman looks good

File: comms/Autoupdate.cpp [hunk 1]
```diff
@@ -146,3 +146,33 @@ IAutoUpdate * CreateIAutoUpdate()
 {
     return new CAutoUpdate();
 }
+
+COMMS_API bool DNSLookup(std::_tstring & url) {
+    size_t found = url.find_first_of(_T(":"));
```
Comment (richardkchapman) [inline comms/Autoupdate.cpp]: Might be clearer/safer if this said "://" rather than ":" (assuming I am understanding the purpose correctly)

**Comment by richardkchapman** [comment]
@GordonSmith One comment

**Comment by GordonSmith** [comment]
@richardkchapman changes made as suggested

**Pull request merged**